AssemblerLib/VectorMatrixAssembler.h

```diff
@@ -136,16 +136,20 @@ class VectorMatrixAssembler<GlobalMatrix, GlobalVector, LocalAssembler,
     /// \attention The index \c id is not necessarily the mesh item's id.
     void preTimestep(std::size_t const id,
                      LocalAssembler& local_assembler,
-                     GlobalVector const& x) const
+                     GlobalVector const& x,
+                     double const t,
```

Comment:
I'd prefer two doubles for the time being. However, then one could also discuss their order. Cf. assemble(t,x,M,K,b) vs preTimestep(x,t,dt).